### PR TITLE
Update Tapir in docker images

### PR DESCRIPTION
**Pull request by prigoyal**
cc @wsmoses there you go, once this passes CI, I am also building conda packages atm and will send out a PR separate for that.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: ~/TensorComprehensions
     resource_class: xlarge
     docker:
-      - image: tensorcomprehensions/linux-trusty-gcc4.8-tapir5.0-cuda8-cudnn6-py3-conda:x86
+      - image: tensorcomprehensions/linux-trusty-gcc4.8-tapir5.0-cuda8-cudnn6-py3-conda:x86_1
```
Comment (wsmoses) [inline .circleci/config.yml]: Is there a reason we're not updating the current image instead of making a new one?

Comment (prigoyal) [reply .circleci/config.yml]: yep, the current one is stable and is being used by users as well. I will need to sync everything with conda packages, python stuff etc. to be able to switch without hazards. I did see some flakiness earlier with new image that i couldn't repro. other than that, no reason :)

**Comment by prigoyal** [comment]
can I hazzz a stamp please @wsmoses ? :)

**Comment by prigoyal** [comment]
thanks @ftynse I'll wait for @wsmoses for few minutes if he feels happy with this, otherwise will land this.

**Pull request merged**